script/dev

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+'use strict'
+
+// Run bootstrap first to ensure all the dependencies used later in this script
+// are installed.
+require('./bootstrap')
+
+const path = require('path')
+const glob = require('glob')
+const colors = require('colors')
+const CONFIG = require('./config')
+const {spawn} = require('child_process')
+const runApmInstall = require('./lib/run-apm-install')
+
+// Install the local core packages in-place so they can be used in dev mode
+var files = glob.sync(path.join(CONFIG.repositoryRootPath, 'packages/*/package.json'))
+if (files.length > 0) {
+  console.log('\nInstalling core packages for use in dev mode...')
+
+  files.forEach(file => {
+    const packageDir = path.dirname(file)
+    process.stdout.write(`Installing packages/${path.basename(packageDir)} `)
+    runApmInstall(path.dirname(file), false, true)
```

Comment:
I'm suppressing apm's output here so that I can write my own similar string for each package so that we don't see `Installing modules...` 50 times in a row :)

Reply:
Also realized I should have used packageDir in the first argument here, will fix it